tests/Integration/content-helper/ContentHelperPostListStatsTest.php

```diff
@@ -719,7 +719,15 @@ public function test_script_of_parsely_stats_admin_column_on_valid_posts_and_val
 		global $wp_scripts;
 
 		ob_start();
-		var_dump( $wp_scripts->print_inline_script ( 'post-list-stats-script', 'before' ) ); // phpcs:ignore
+
+		global $wp_version;
+		if ( true === version_compare( substr( $wp_version, 0, 3 ), '6.3', '>=' ) ) {
+			// @phpstan-ignore-next-line
```

Comment:
The PHPStan comment is because PHPStan complains that we're calling an undefined function. We will probably be able to remove it in the future.